### ISSUE TITLE
create a code-snippet for a function stateless component

### ISSUE THREAD

**Issue by devMozao** [eruptionjs/core#13]
create a new typescriptreact code snippet to facilitate the write of a functional component, something like

```jsx
export const Component: FC = () => {
//
}

export default Component
```

also, update all the current snippets to be accessed with abbreviations like

abbreviation: `uef`
meaning: `use effect function (in order to remember)`
the output: `const [foo, setFoo] = useState();`

**Comment by kfurfles** [comment]
I create three variations of component template inspired on CRE TS Template.
Can you validate the initial structure of them ?

**Without Memo**

```typescript
/**
*
* SimpleTest
*
*/
import * as React from 'react';

interface Props {}

export const SimpleTest: React.FC<Props> = (props) => {
  return <div>Component: SimpleTest</div>
};

```

**Only Memo**

```typescript
/**
*
* MemoAndTest
*
*/
import React, { memo } from 'react';

interface Props {}

export const MemoAndTest: React.FC<Props> = memo((props) => {
  return <div>Component: MemoAndTest</div>
});
```

**Test**

```typescript
import * as React from 'react';
import { render } from '@testing-library/react';

import { MemoAndTest } from '..';

describe('<MemoAndTest  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<MemoAndTest />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
```

Please comment your considerations, if needed, I'ill fix them and create PR

**Comment by devMozao** [comment]
@kfurfles Sorry about the confusion. I did a poor job explaining the task. It was about creating and editing code snippets inside the `typescriptreact.code-snippets.json` file inside the `.vscode` folder.

I just pushed these new snippets, eliminating the urge to have a snippet plugin on vscode itself.

If you want, feel free to test it. It's already merged with `main` branch.

Closed with: 577a270
